reference/docs-conceptual/dev-cross-plat/create-help-using-platyps.md

```diff
@@ -0,0 +1,297 @@
+
+   ```powershell
+   Import-Module <your module name>
+   ```
+
+1. Use PlatyPS to generate Markdown files for your module landing page and all associated cmdlets
+   within the module. Repeat this step for each module you need to document.
+
+   ```powershell
+   $OutputFolder = <output path>
+   $parameters = @{
+       Module = <ModuleName>
+       OutputFolder = $OutputFolder
+       AlphabeticParamsOrder = $true
+       WithModulePage = $true
+       ExcludeDontShow = $true
+       Encoding = 'UTF8BOM'
+   }
+   New-MarkdownHelp @parameters
+
+   New-MarkdownAboutHelp -OutputFolder $OutputFolder -AboutName "topic_name"
+   ```
+
+   If the output folder does not exist, `New-MarkdownHelp` creates it. In this example,
+   `New-MarkdownHelp` creates a Markdown file for each cmdlet in the module. It also creates a file
+   named `<ModuleName>.md`. This file contains a list of the cmdlets contained in the module and
+   placeholders for the **Synopsis** description. `New-MarkdownAboutHelp` creates a new _about_ file
+   named `about_topic_name.md`.
+
+   For more information, see [New-MarkdownHelp][] and [New-MarkdownAboutHelp][].
+
+### Update existing Markdown content when the module changes
+
+PlatyPS can also update existing Markdown files for a module. Use this step to update existing
+modules that have new cmdlets, new parameters, or parameters that have changed.
+
+1. Import your new module into the session. Repeat this step for each module you need to document.
+
+   Run the following command to import your modules:
+
+   ```powershell
+   Import-Module <your module name>
+   ```
+
+1. Use PlatyPS to update Markdown files for your module landing page and all associated cmdlets
+   within the module. Repeat this step for each module you need to document.
+
+   ```powershell
+   $parameters = @{
+       Path = <folder with Markdown>
+       RefreshModulePage = $true
+       AlphabeticParamsOrder = $true
+       UpdateInputOutput = $true
+       ExcludeDontShow = $true
+       LogPath = <path to store log file>
+       Encoding = 'UTF8BOM'
+   }
+   Update-MarkdownHelpModule @parameters
+   ```
+
+   `Update-MarkdownHelpModule` updates the cmdlet and module Markdown files in the specified folder.
+   It does not update the `about_*.md` files. The module file (`ModuleName.md`) receives any new
+   **Synopsis** text that has been added to the cmdlet files. Updates to cmdlet files include the
+   following change:
+
+   - New parameter sets
+   - New parameters
+   - Updated parameter metadata
+   - Updated input and output type information
+
+   For more information, see [Update-MarkdownHelpModule][].
+
+## Edit the new or updated Markdown files
+
+PlatyPS documents the syntax of the parameter sets and the individual parameters. It can't create
+descriptions or provide examples. The specific areas where content is needed are contained in curly
+braces. For example: `{{*Fill in the Synopsis*}}`
```

Comment:
I wonder if an image of the file with highlights on {{*Fill in the ...}} would help visualize the edits needed.

Reply:
I will look into this.